src/components/node-list/index.js

```diff
@@ -28,12 +28,6 @@ const isModularPipelineType = (type) => type === 'modularPipeline';
 /**
  * Provides data from the store to populate a NodeList component.
  * Also handles user interaction and dispatches updates back to the store.
- *
- * The data are hierarchical but provided through flat lists in the form of:
```

Comment:
This part of the description can be deleted as it is not relevant to the new tree list anymore